.github/workflows/build.yml

```diff
@@ -214,6 +214,17 @@ jobs:
         shell: bash
         run: parallelize results Build-Executor
 
+      - # === Scan for CVEs (Linux only) ===
+        name: Scan for CVEs
+        if: runner.os == 'Linux'
```

Comment:
We will need to scan for all platforms. It's ok if this runs ON linux, so long as we verify all binaries for all platforms.

Additionally, we should only run this on release branches and the nightly I think, no point slowing down regular PRs.

We could add a new job that runs in between build.yml and release.yml. Because we are already transferring all platform bits from build to release.yml, so we could inject something in the middle there for CVE validation.

Reply:
Well, that's what I was going to say...

For additional information you can run `go version -m <executable> ` on a Go binary to see that there are differences in the modules used. It doesn't appear that versions are different but the Windows build may include modules that Linux does not and vice versa.

It also appears that Trivy has releases for MacOS and Windows, if that helps: https://github.com/aquasecurity/trivy/releases

Reply:
Very well, I've added a new Scan action inbetween build and deploy that scans all platform-specific binaries in one go. I disagree that it should only run on release branches and nightlies. PRs should not introduce CVEs only for them to be caught later. We should be proactive.

Reply:
@MDrakos did I hijack your review? 😅 Not my intention. When I reviewed this I was under the impression I was flagged, maybe I got my wires crossed.